### PR TITLE
Render 0.18 changelog using towncrier

### PR DESCRIPTION
**Pull request by maxnoe**
While this is automatically created, please have a look if this is what we want it to be.

### CODE REVIEW

**Comment by maxnoe** [comment]
I also ran https://github.com/maxnoe/test_ctapipe_ml/ again and results look good, I'd say we are ready for release

File: CHANGES.rst [hunk 1]
```diff
@@ -1,3 +1,189 @@
...
+  and ``astropy.coordinates.EarthLocation``, enabling the conversion
+  between relative array coordinates (used in the simulation) and
+  absolute real-world coordinates. [`#2167 <https://github.com/cta-observatory/ctapipe/pull/2167>`__]
+
+- The ``ctapipe-display-dl1`` tool now has a ``QualityQuery`` instance which can be used
+  to select which images should be displayed. [`#2172 <https://github.com/cta-observatory/ctapipe/pull/2172>`__]
+
+- Add a new ``ctapipe.io.HDF5Merger`` component that can selectively merge
+  HDF5 files produced with ctapipe. The new component is now used in the
+  ``ctapipe-merge`` tool but can also be used on its own.
```
Comment (LukasNickel) [inline CHANGES.rst:127]: We could mention its also used in `apply-models`

File: CHANGES.rst [hunk 2]
```diff
@@ -1,3 +1,189 @@
...
+  implementations via the ``ctapipe_reco`` entry_point. [`#2101 <https://github.com/cta-observatory/ctapipe/pull/2101>`__]
+
+- Migrate muon analysis into the ``ctapipe-process`` tool:
+
+  1. The former ``muon_reconstruction`` tool is dropped and all functionalities are transferred 
+     into the ``ctapipe-process`` tool.
+
+  2. The ``process`` tool now has a ``write_muon_parameters`` flag which defaults to ``false``.
+     Muons are only analyzed and written if the flag is set. Analyzing muons requires DL1b image 
+     parameters so even if ``write_parameters`` is ``false`` and no parameters are present in the
```
Comment (LukasNickel) [inline CHANGES.rst:20]: I think this sentence is a bit confusing. Even if they are not available, they are computed?

File: CHANGES.rst [hunk 3]
```diff
@@ -1,3 +1,189 @@
...
+
+  2. The ``process`` tool now has a ``write_muon_parameters`` flag which defaults to ``false``.
+     Muons are only analyzed and written if the flag is set. Analyzing muons requires DL1b image 
+     parameters so even if ``write_parameters`` is ``false`` and no parameters are present in the 
+     ``EventSource`` those are computed. 
+
+  3. Two instances of ``QualityQuery``, ``MuonProcessor.ImageParameterQuery`` and ``MuonProcessor.RingQuery`` 
+     are added to the muon analysis to either preselect images according to image parameters and 
+     to select images according to the initial, geometrical ring fit for further processing. 
+     Deselected events or those where the muon analysis fails are being returned and written
```
Comment (LukasNickel) [inline CHANGES.rst:26]: I also dislike this sentence.
`Deselected events and those where the muon analysis fails, are written out as nan` ?

File: CHANGES.rst [hunk 4]
```diff
@@ -1,3 +1,189 @@
...
+  in ``ctapipe.image.muon.ring_completeness``, which raises an error with
+  astropy 5.2.1. [`#2197 <https://github.com/cta-observatory/ctapipe/pull/2197>`__]
+
+
+Maintenance
+-----------
+
+- Use towncrier for the generation of change logs [`#2144 <https://github.com/cta-observatory/ctapipe/pull/2144>`__]
+
+- Replace usage of deprecated aastropy matrix function. [`#2166 <https://github.com/cta-observatory/ctapipe/pull/2166>`__]
```
Comment (LukasNickel) [inline CHANGES.rst:168]: astropy

File: CHANGES.rst [hunk 5]
```diff
@@ -1,3 +1,189 @@
...
+     If the Tool must fail and only the non-zero error case is important to test,
+     set ``raises=False`` (as it was before).
+
+  2. If the ``cwd`` parameter is ``None`` (as per default), now a temporary directory
+     is used instead of the directory, where ``run_tool`` is called (typically via
+     pytest). This way, log-files and other output files don't clutter your
+     working space. [`#2175 <https://github.com/cta-observatory/ctapipe/pull/2175>`__]
+
+- Remove ``-f`` flag as alias for ``--overwrite`` and fail early if output exists, but overwrite is not set [`#2213 <https://github.com/cta-observatory/ctapipe/pull/2213>`__]
+
```
Comment (LukasNickel) [inline CHANGES.rst:49]: Remove one empty line

File: CHANGES.rst (hunk 1 above)
Comment (maxnoe) [reply CHANGES.rst:127]: done

File: CHANGES.rst (hunk 2 above)
Comment (maxnoe) [reply CHANGES.rst:20]: Yes.

Comment (maxnoe) [reply CHANGES.rst]: Updated.

File: CHANGES.rst (hunk 3 above)
Comment (maxnoe) [reply CHANGES.rst:26]: changed to "filled with invalid value markers"

**Review by LukasNickel — DISMISSED** [review]

**Review by LukasNickel — APPROVED** [review]

**Review by nbiederbeck — APPROVED** [review]

File: CHANGES.rst [hunk 6]
```diff
@@ -1,3 +1,196 @@
+Ctapipe v0.18.0 (2023-02-09)
```
Comment (kosack) [inline CHANGES.rst:1]: is there any way to remove the capitalization here? Otherwise look good!

Comment (maxnoe) [reply CHANGES.rst:1]: sure

**Review by kosack — APPROVED** [review]

**Review by nbiederbeck — APPROVED** [review]

**Pull request merged**